test.yaml

```diff
@@ -0,0 +1,13 @@
+http:
```

Comment:
Can we add test file similar to https://github.com/flanksource/konfigadm/blob/master/test/setup_test.go#L161 and then move this files into a test or fixtures folder?


We probably also need to split into 2 sets of fixtures, 1 for passing checks and 1 for failing.